src/libpriv/rpmostree-passwd-util.c

```diff
@@ -1182,10 +1057,7 @@ rootfs_has_usrlib_passwd (int rootfs_dfd,
           return TRUE;
         }
       else
-        {
-          glnx_set_error_from_errno (error);
-          return FALSE;
-        }
+        return glnx_throw_errno_prefix (error, "fstatat");
```

Comment:
Can we add the `(%s)` arg to these errno prefix calls? (This one and the ones below; maybe just for those that don't use hardcoded paths or `glnx_strjoina` for now at least). Just to make narrowing down these types of things easier.